src/caret_analyze/plot/graphviz/chain_latency.py

```diff
@@ -99,21 +99,46 @@ def get_attr_node(
     lstrip_s: float,
     rstrip_s: float
 ) -> GraphAttr:
+    def calc_latency_from_path_df(target_columns: list[str]) -> np.ndarray:
+        target_df = path.to_dataframe(
+            remove_dropped=remove_dropped,
+            treat_drop_as_delay=treat_drop_as_delay,
+            lstrip_s=lstrip_s,
+            rstrip_s=rstrip_s,
+        )[target_columns]
+        source_stamps_ns = np.array(target_df.iloc[:, 0].values)
+        dest_stamps_ns = np.array(target_df.iloc[:, -1].values)
+        latency_ns = dest_stamps_ns - source_stamps_ns
+        if remove_dropped:
+            latency_ns = latency_ns.astype('int64')
+        return latency_ns
+
     graph_nodes: List[GraphNode] = []
     remove_dropped = False
 
-    for node_path in path.node_paths:
+    for i, node_path in enumerate(path.node_paths):
         node_name = node_path.node_name
         label = node_name
 
-        if node_path.column_names != []:
+        if i == 0 and path.include_first_callback:
+            first_cb_columns = path.column_names[0:2]
+            latency = calc_latency_from_path_df(first_cb_columns)
+            label += '\n' + to_label(latency)
+
+        elif i == len(path.node_paths)-1 and path.include_last_callback:
+            last_cb_columns = path.column_names[-2:]
+            latency = calc_latency_from_path_df(last_cb_columns)
```

Comment:
Same as above.
```suggestion
            latency = _calc_latency_from_path_df(last_cb_columns)
```

Reply:
Because `calc_latency_from_path_df()` is an intra-functional function, it is kept secret from the outside world without `_`.